docs/validators/redundancy.md

```diff
@@ -0,0 +1,48 @@
+---
+id: redundancy
+title: Node redundancy
+---
+
+Each validator node can be backed up by one or several backup nodes in order to 
+provide security of the network and to assure that a node will unlikely get jailed.
+
+The redundancy mechanism relies on the following principle: the owner of the BLS key will start `n` nodes
+with almost the same configuration, and the same `validatorKey.pem` file. The difference between 
+configurations consists on an option inside the `prefs.toml` file.
+That configuration option called `RedundancyLevel` can be used as following:
+
+- a 0 value will represent that the node is the main validator.
+The value 0 will be the default, if the option is missing (as we do not re-write the `prefs.toml` 
+  files during the node's upgrade) will still make that node the main validator: 
+  backwards compatibility holds.
+- a positive value will represent the "order of the back-up node" in the automatic fail-over sequence. 
+  Example: let's suppose we have 3 nodes running with the same key. One with the redundancy level set to 0,
+  one with 1 and one with 3. The node with level 0 will propose and sign upon blocks. The other 2 will 
+  sync (and shuffle out) but will not sign anything. If the main validator fails, the back-up validator 
+  with level 1 will start producing/signing blocks after `level*5` missed rounds. So, after 5 
+  missed rounds by the main validator, the back-up node with level 1 will take the turn. 
+  Now, if the back-up node 1 is down, the second back-up node will start producing/signing blocks 
```

Comment:
Replace:
Now, if the back-up node 1 is down, the second back-up node will start producing/signing blocks 

with
If hot-standby node 1 is down as well, hot-standby node 2 will step in

Reply:
done